starboard/linux/x64x11/run_starboard_main.cc

```diff
@@ -14,9 +14,62 @@
 
 #include "starboard/event.h"
 
+#include <malloc.h>
+#include <time.h>
+
+#include "starboard/configuration.h"
+#include "starboard/crashpad_wrapper/wrapper.h"
+#include "starboard/event.h"
+#include "starboard/shared/signal/crash_signals.h"
+#include "starboard/shared/signal/debug_signals.h"
+#include "starboard/shared/signal/suspend_signals.h"
+#include "starboard/shared/x11/application_x11.h"
+#if SB_IS(EVERGREEN_COMPATIBLE)
+#include "starboard/common/command_line.h"
+#include "starboard/common/paths.h"
+#include "starboard/elf_loader/elf_loader_constants.h"
+#endif
 #include "starboard/shared/x11/application_x11.h"
 
 int SbRunStarboardMain(int argc, char** argv, SbEventHandleCallback callback) {
```

Comment:
Should linux/raspi's main (defined in `//<platform_path>/main.cc`) be changed to remove all of this code? Right now I think it would be repeated for i.e. non-modular linux, right?